python/mujinwebstackclient/webstackclient.py

```diff
@@ -1064,22 +1067,28 @@ def Backup(self, saveconfig=True, savemedia=True, backupscenepks=None, saveapps=
             'apps': 'true' if saveapps else 'false',
             'itl': 'true' if saveitl else 'false',
             'detection': 'true' if savedetection else 'false',
+            'state': 'true' if savestate else 'false',
             'calibration': 'true' if savecalibration else 'false',
             'debug': 'true' if savedebug else 'false',
+            'eds': 'true' if saveeds else 'false',
+            'iodd': 'true' if saveiodd else 'false',
             'backupScenePks': ','.join(backupscenepks) if backupscenepks else None,
         }, timeout=timeout)
         if response.status_code != 200:
             raise WebstackClientError(response.content.decode('utf-8'), response=response)
         return response
 
-    def Restore(self, file, restoreconfig=True, restoremedia=True, restoreapps=True, restoreitl=True, timeout=600):
+    def Restore(self, file, restoreconfig=True, restoremedia=True, restoreapps=True, restoreitl=True, restorestate=True, restoreeds=True, restoreiodd=True, timeout=600):
```

Comment:
Let's not add `restorestate` option here, because webstack does not support resotre state at the moment.

Reply:
We will support backup/resotre redis state in the future

Reply:
done